src/BizHawk.Client.EmuHawk/tools/TAStudio/TAStudio.ListView.cs

```diff
@@ -999,24 +999,42 @@ private void TasView_MouseWheel(object sender, MouseEventArgs e)
 			}
 		}
 
+		public void SetMarker(int frame = -1)
+		{
+			if (frame == -1) frame = Emulator.Frame;

```

Comment:
Is `-1` expected via `TasView.CurrentCell.RowIndex`, or is this just to handle the default argument? If it's the latter, you should use `(int?) null`, or have a separate overload `public void SetMarker() => SetMarker(Emulator.Frame);`